catch_analysis_tools/astrometry.py

```diff
@@ -0,0 +1,435 @@
+import os
+import subprocess
+import json
+import requests
+import numpy as np
+import pandas as pd
+import sep
+import fitsio
+import matplotlib.pyplot as plt
+from astropy.io import fits
+from astropy.wcs import WCS
+from astropy.table import Table
+from astropy.coordinates import SkyCoord
+import calviacat as cvc
+
+def load_fits_image(target_name, data_sources, fn=3):
```

Comment:
This function is something we don't need in the repository, since the catch query will be run by the user before the analysis tools are used.